src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/dto/TestResultsDTO.java

```diff
@@ -146,15 +150,17 @@ public List<CommitDTO> getCommits() {
         return commits;
     }
 
-    public List<TestSuiteDTO> getResults() {
-        return results;
+    public List<BuildJobDTOInterface> getResults() {
+        return new ArrayList<>(results);
     }
```

Comment:
Why do you need getResults, if you have getBuildJobs

Reply:
I just left `getResults()` as it is the getter for the corresponding attribute and JSON property. Actually, we don't need it, but I thought it would make it more consistent. Anyway, I can remove it if you want. I'm not sure the additional getter it's really necessary or just confusing.